eng/Version.Details.xml

```diff
@@ -24,6 +24,10 @@
       <Uri>https://github.com/dotnet/command-line-api</Uri>
       <Sha>803d8598f98fb4efd94604b32627ee9407f246db</Sha>
     </Dependency>
+    <Dependency Name="System.Text.Json" Version="6.0.9">
```

Comment:
Note that this is the same version information that dotnet/aspnetcore has for its release/6.0 branch: https://github.com/dotnet/aspnetcore/blob/release/6.0/eng/Version.Details.xml#L236